tqdm/_tqdm.py

```diff
@@ -948,6 +948,7 @@ def __iter__(self):
                                 if avg_time is None \
                                 else smoothing * delta_t / delta_it + \
                                 (1 - smoothing) * avg_time
```

Comment:
here, ``avg_time`` is useless because it has never been read. Actually, ``avg_time`` should be written back to ``self.avg_time``